packages/components/src/internal/components/forms/detail/DetailDisplay.tsx

```diff
@@ -94,11 +95,14 @@ function processFields(
     titleRenderer?: TitleRenderer,
     options?: RenderOptions,
     fileInputRenderer?: (col: QueryColumn, data: any) => ReactNode,
-    onAdditionalFormDataChange?: (name: string, value: any) => any
+    onAdditionalFormDataChange?: (name: string, value: any) => any,
+    internalSpacesWarningFieldKeys?: string[]
 ): OrderedMap<string, DetailField> {
     return queryColumns.reduce((fields, c) => {
         const fieldKey = c.fieldKey.toLowerCase();
-
+        if (internalSpacesWarningFieldKeys?.indexOf(fieldKey) > -1) {
```

Comment:
Are `internalSpacesWarningFieldKeys` always in lower cases?

Reply:
Yes.